.github/workflows/mobile.yml

```diff
@@ -0,0 +1,97 @@
+name: Mobile Deployment
+on: [deployment_status,   workflow_dispatch]
+
+jobs:
+    # TODO move this to a common file
```

Comment:
Was this `TODO` completed by this PR? Or is the TODO to pull this common `Setup` stuff out to another file?